components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/AssetBlob.java

```diff
@@ -35,6 +36,8 @@
 {
   private final BlobRef blobRef;
 
+  private final Blob blob;
```

Comment:
@mrprescott thanks for the explanation - I think it's worthwhile capturing that detail in a backlog issue (or in a comment if there's already an issue for the node<->blob work)